src/Host/UserProfile/Impl/ServiceLogger.cs

```diff
@@ -28,23 +27,41 @@ internal sealed class ServiceLogger : ILogger, IDisposable {
             return true;
         }
 
+        
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter) {
-            var writer = _writer;
-            if (writer == null) {
-                return;
-            }
-
-            string message = formatter(state, exception);
-            lock (writer) {
-                writer.WriteLine("[{0:u}] <{1}> ({2}):", DateTime.Now, _category, logLevel.ToString()[0]);
-                writer.WriteLine(message);
-
-                if (exception != null) {
-                    writer.WriteLine("Exception: " + exception);
+            lock (_logLock) {
+                string message = formatter(state, exception);
+                using (EventLog eventLog = new EventLog("Application")) {
+                    EventLogEntryType logType = GetLogType(logLevel);
+                    eventLog.Source = "Application";
+                    string logMessage;
+                    if (exception != null) {
+                        logMessage = string.Format("[{0:u}] <{1}> ({2}), Message: {3}, Exception: {4}", DateTime.Now, _category, logLevel.ToString()[0], message, exception);
```

Comment:
Since message and exception details are both likely to be long, and possibly multiline, I would suggest using newlines instead of commas here, and dropping "Message". So that the output would be something like this:
```
[10:11:12] <Component> (I):
Something bad happened

Exception: InvalidOperationException in .. .at ...
  at ...
  at ..
```